docs/ref/models/lookups.txt

```diff
@@ -237,6 +237,8 @@ the following methods:
         can be used for compiling vendor specific SQL. If ``lhs`` is not
         ``None``, use it as the processed ``lhs`` instead of ``self.lhs``.
 
+        .. versionadded:: 6.1
+
```

Comment:
```suggestion
```
This existed for Lookup prior to this change so we don't add this here

What I meant would have been something like this:
```diff
--- a/docs/ref/models/lookups.txt
+++ b/docs/ref/models/lookups.txt
@@ -180,6 +180,13 @@ the following methods:
         :class:`~django.db.models.Field` instance. By default is the same as
         its ``lhs.output_field``.
 
+    .. method:: process_lhs(compiler, connection, lhs=None)
+
+        .. versionadded:: 6.1
+
+        Same as :meth:`~Lookup.process_lhs`. Returns a tuple
+        ``(lhs_string, lhs_params)``, as returned by ``compiler.compile(lhs)``.
```